client/gutenberg/editor/components/header/more-menu/index.js

```diff
@@ -0,0 +1,62 @@
+/** @format */
+/**
+ * External dependencies
+ */
+import React, { Fragment } from 'react';
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { IconButton, Dropdown, MenuGroup } from '@wordpress/components';
+
+/**
+ * Internal dependencies
+ */
+import CopyContentMenuItem from '../copy-content-menu-item';
+import KeyboardShortcutsHelpMenuItem from '../keyboard-shortcuts-help-menu-item';
+import ModeSwitcher from '../mode-switcher';
+import OptOutMenuItem from '../opt-out-menu-item';
+import OptionsMenuItem from '../options-menu-item';
+import PluginsMoreMenuGroup from '../plugins-more-menu-group';
+import ToolsMoreMenuGroup from '../tools-more-menu-group';
+import WritingMenu from '../writing-menu';
+
+const ariaClosed = __( 'Show more tools & options' );
+const ariaOpen = __( 'Hide more tools & options' );
+
+/* eslint-disable wpcalypso/jsx-classname-namespace */
+const MoreMenu = () => (
+	<Dropdown
+		className="edit-post-more-menu"
+		contentClassName="edit-post-more-menu__content"
+		position="bottom left"
+		renderToggle={ ( { isOpen, onToggle } ) => (
+			<IconButton
+				icon="ellipsis"
+				label={ isOpen ? ariaOpen : ariaClosed }
+				onClick={ onToggle }
+				aria-expanded={ isOpen }
+			/>
+		) }
+		renderContent={ ( { onClose } ) => (
+			<Fragment>
+				<WritingMenu onClose={ onClose } />
+				<ModeSwitcher onSelect={ onClose } />
+				<PluginsMoreMenuGroup.Slot fillProps={ { onClose } } />
+				<MenuGroup label={ __( 'Tools' ) }>
```

Comment:
Why don't we render this group just with `ToolsMoreMenuGroup` as [Gutenberg does](https://github.com/WordPress/gutenberg/blob/2a66db0fc99a9fb1ba99f61e59e6a0b2a4a5f9ef/packages/edit-post/src/components/header/more-menu/index.js#L39)?

It could be more complex to maintain when we decide to incorporate any future update from Gutenberg if we forget that we have some differences.

Reply:
Actually, this `MoreMenu` needs to be different because of the "Switch to Classic" option, so maybe it is not a big deal if we render the items differently.

Reply:
I've changed the implementation of `ToolsMoreMenuGroup` to not be a container for the Tools section (because I had to reimplement the items inside it), but still be a slot to be filled externally.